src/main/drivers/accgyro_mpu.c

```diff
@@ -224,60 +227,77 @@ bool mpuCheckDataReady(gyroDev_t* gyro)
 }
 
 #ifdef USE_SPI
-static bool detectSPISensorsAndUpdateDetectionResult(gyroDev_t *gyro)
+static bool detectSPISensorsAndUpdateDetectionResult(gyroDev_t *gyro, const sensorBus_t *bus)
 {
 #ifdef USE_GYRO_SPI_MPU6000
-    if (mpu6000SpiDetect()) {
+#ifdef MPU6000_CS_PIN
+    gyro->bus.spi.csnPin = bus->spi.csnPin == IO_NONE ? IOGetByTag(IO_TAG(MPU6000_CS_PIN)) : bus->spi.csnPin;
```

Comment:
Agreed it is not necessary. However I think using it, overall, makes the code slightly clearer.